tests/test_import.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import unittest
+
+from huggingface_hub.file_download import is_tf_available
+
+
+def require_tf(test_case):
+    """
+    Decorator marking a test that requires TensorFlow.
+
+    These tests are skipped when TensorFlow is not installed.
+
+    """
+    if not is_tf_available():
+        return unittest.skip("test requires Tensorflow")(test_case)
+    else:
+        return test_case
+
+
+@require_tf
```

Comment:
Actually, will this test ever be executed? If it uses `require_tf`, i will never run. The only test that builds tensorflow is ` pytest --cov -Werror::FutureWarning -sv ./tests/test_keras*` at https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/python-tests.yml. I think we need to remove the decorator?